lua/lspconfig/configs/nextflow_ls.lua

```diff
@@ -0,0 +1,35 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'java', '-jar', 'nextflow-language-server-all.jar' },
+    filetypes = { 'nextflow' },
+    root_dir = util.root_pattern('nextflow.config', '.git'),
+    settings = {
+      nextflow = {
+        files = {
+          exclude = { '.git', '.nf-test', 'work' },
+        },
+      },
+    },
+  },
+  docs = {
+    description = [[
+https://github.com/nextflow-io/language-server
+
+Requirements:
+ - Java 17+
+
+`nextflow_ls` can be installed by following the instructions [here](https://github.com/nextflow-io/language-server#development).
+
+If you have installed nextflow language server, you can set the `cmd` custom path as follow:
+
+```lua
+require'lspconfig'.nextflow_ls.setup{
+    cmd = { "java", "-jar", "path/to/nextflow_ls/language-server-all.jar" },
+    ...
```

Comment:
```suggestion
    cmd = { 'java', '-jar', 'nextflow-language-server-all.jar' },
    filetypes = { 'nextflow' },
    root_dir = util.root_pattern('nextflow.config', '.git'),
    settings = {
      nextflow = {
        files = {
          exclude = { '.git', '.nf-test', 'work' },
        },
      },
    },
```

Reply:
Is this correct? The user doesn't need to fully specify the file types and things when they are simply setting up the language server. They just need to make sure to potentially update the command with the path to the jar. This doc string was also basically a direct copy of the groovy_ls doc string in this repository.

Reply:
Ah, I see! Thanks so much for the clarification

Reply:
Just for reference, a follow up PR here is probably a good idea to apply that methodology as well: https://github.com/neovim/nvim-lspconfig/blob/52302604e3c667cfdf33aadf89088e96eb3c5da3/lua/lspconfig/configs/groovyls.lua#L31